### ISSUE TITLE
Python version restricted to >=3.9, inability to use official docker containers of tensorflow

### ISSUE THREAD

**Issue by xehartnort** [tensorflow/privacy#472]
Since the addition of [dp-accounting](https://pypi.org/project/dp-accounting/), the minimum version of python required to use the latest releases of this package is restricted to >=3.9. This restriction causes plenty of inconsistencies when using the [official docker images for tensorflow](https://hub.docker.com/r/tensorflow/tensorflow/) and the [official nvidia tensorflow images](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow), given that they only support python3.8.

To sum up, it is not possible to use the latest version of tensorflow-privacy with the official docker containers of tensorflow due to the python version required. 

Consequently, it will be nice to add a note in the README indicating that the python version required is >=3.9.

**Comment by minwoo33park** [comment]
This could be a workaround.
https://cloud.google.com/tensorflow-enterprise/docs/overview
https://pantheon.corp.google.com/gcr/images/deeplearning-platform-release
